keps/sig-aws/20190501-aws-nlb-beta.md

```diff
@@ -0,0 +1,88 @@
+---
+title: graduate-aws-nlb-to-beta
+authors:
+  - "@M00nF1sh"
+owning-sig: sig-aws
+participating-sigs: null
+reviewers:
+  - "@dnishi"
+approvers:
+  - "@justinsb"
+editor: TBD
+creation-date: 2019-05-01
+last-updated: 2019-05-01
+status: implementable
+see-also:
+  - 
+replaces:
+superseded-by:
+---
+
+# Graduate AWS Network Load Balancer Support to beta
+
+## Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+- [Proposal](#proposal)
+  - [User Stories](#user-stories)
+- [Design](#design)
+  - [Test Plan](#test-plan)
+    - [Needed Tests](#needed-tests)
+  - [Graduation Criteria](#graduation-criteria)
+- [Proposed roadmap](#proposed-roadmap)
+  - [1.15](#115)
+  - [1.16](#116)
+  - [1.18](#118)
+- [Implementation History](#implementation-history)
+
+## Summary
+
+[AWS Network Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/introduction.html) is an L4 Load Balancer on AWS platform, which can be used as implementation for Service with LoadBalancer type.
+
+## Motivation
+
+[AWS Network Load Balancer](https://aws.amazon.com/blogs/opensource/network-load-balancer-support-in-kubernetes-1-9/) has been supported in Kubernetes as Alpha feature since v1.9. Since then, we have stablized the code and API, and decide to graduate it to Beta. 
+
+### Goals
+
+* Promote AWS Network Load Balancer support to beta version.
+
+## Proposal
+
+### User Stories
+* As an application developer, I want to be able to use AWS Network Load balancer for Service with Load Balancer type in production environment.
```

Comment:
instead of "As an application developer, I want to be able to use AWS Network Load balancer for Service with Load Balancer type in production environment."

"An application developer or infrastructure engineer, wants to use AWS Network Load balancer to declare their Kubernetes Service with type: Load Balancer for their development, staging or production Kubernetes environments running in AWS cloud."